xbmc/platform/posix/XTimeUtils.cpp

```diff
@@ -8,216 +8,10 @@
 
 #include "utils/XTimeUtils.h"
 
-#include <errno.h>
-#include <mutex>
-#include <time.h>
-
-#include <sys/times.h>
-
-#if defined(TARGET_ANDROID) && !defined(__LP64__)
-#include <time64.h>
-#endif
-
-#define WIN32_TIME_OFFSET ((unsigned long long)(369 * 365 + 89) * 24 * 3600 * 10000000)
-
 namespace KODI
```

Comment:
Would it be possible to remove this file completely?

`xbmc/platform/posix/XTimeUtils.cpp`